packages/db/src/index.ts

```diff
@@ -1,2 +1,5 @@
 export { getKnexClient } from './connection';
 export { getKnexConfig, localStackConnectionEnv } from './config';
+export * as Executions from './Executions';
+export * as database from './database';
```

Comment:
super nit/discuss: do we want the reference to be database.getDbClient because of the splat export?  

If so should we make the methods getClient and getTransaction

Reply:
I'm fine with not doing a splat export if the names `getDbClient`, `getDbTransaction` work for you

Reply:
I think my only nit here was the redundant nature of having Db in a database.* function.   I'm fine with either database.getClient or getDbClient